torch/utils/_sympy/value_ranges.py

```diff
@@ -261,31 +261,36 @@ def exp(x):
 
     @staticmethod
     def log(x):
+        if x.lower <= 0:
```

Comment:
if this is has free symbol it would throw. but we can handle that when we get here